src/functions.ts

```diff
@@ -73,52 +73,35 @@ const createFunctionFile = async (
   fnId: string,
   fnFilePath: string,
 ) => {
-  // Bundle File
-  let isImportMapPresent = false;
-  const importMapPath = `${options.workingDirectory}/import_map.json`;
+  const fnFileRelative = path.join("functions", `${fnId}.js`);
+  const fnBundledPath = path.join(options.outputDirectory, fnFileRelative);
 
+  let denoExecutablePath = "deno";
   try {
-    const { isFile } = await Deno.stat(importMapPath);
-    isImportMapPresent = isFile;
-  } catch (_e) {
-    isImportMapPresent = false;
+    denoExecutablePath = Deno.execPath();
+  } catch (e) {
+    console.log("Error calling Deno.execPath()", e);
```

Comment:
I was thinking we'd fall back to just using `deno` that's on the PATH, but you think we should just hard-fail here? It'd mean we'd be bundling w/ a version that may not be what we expect, so I suppose it might be beneficial to hard-fail? I'm just wary of having no fall-back in the case that something causes that Deno.execPath() to become unreliable...

Reply:
ah gotcha. thats fine w/ me if you expect execPath to fail sometimes

Reply:
tbh I would hope it doesn't, am perhaps traumatized from previous api changes and being overly cautious. I'll add a comment in here at least in case we want to revisit in the future and have confidence in removing the defaulting of it.